spec/presenters/summary/assessment_question_presenter_spec.rb

```diff
@@ -175,8 +163,20 @@
           expect(question).to receive(:parent_group_answer).and_return(false)
         end
 
-        it 'returns No' do
-          expect(presenter.answer).to eq('No')
+        context 'and answer is boolean' do
```

Comment:
The following two contexts have the same [issue](https://github.com/ministryofjustice/moving-people-safely/pull/302/files#r117478537) described above. 

Also they look exactly the same, seems like a good candidate for a shared example.